bats/foreman_helper.bash

```diff
@@ -71,11 +78,12 @@ tCheckPulpYumContent() {
   CONTENT_SOURCE=$1
   BASE_PATH=$2
   LCE=$3
-  RPM_FILE=walrus-0.71-1.noarch.rpm
+  RPM_FILE=$4
```

Comment:
Should this and the next line get defaults, like
```
RPM_FILE=${4:-walrus.blah.rpm}
```

Then you don't need to modify the old tests.

Reply:
Sure, that looks good to me.